src/descriptors/power_system_structs.json

```diff
@@ -1224,7 +1485,57 @@
       },
       {
         "name": "internal",
-        "data_type": "PowerSystems.PowerSystemInternal"
+        "data_type": "PowerSystemInternal"
+      }
+    ],
+    "supertype": "Forecast"
+  },
+  {
+    "struct_name": "ScenarioBased",
```

Comment:
Should we append "Forecast" to all of our Forecast subtypes?  It seems like we could add ScenarioBasedSomethingElse or DeterministicSomethingElse in the future.

Reply:
We could change the name to Scenarios only. I hesitate to call it XXXForecasts since we use it to store realizations as well.

Reply:
But it is a `::Forecast`... if you don't want to call it a XXXForecasts, we should revisit the type definition

Reply:
I am not sure it is necessary to declare another type that won't have an additional functionality.